go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/PaddlePaddle/PaddleFlow
 
-go 1.16
+go 1.18
```

Comment:
It might be better to upgrade 1.18 in next PR.

Reply:
ok